admin-dev/themes/new-theme/js/pages/product/edit/packed-products-manager.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import EntitySearchInput from '@components/entity-search-input';
+import ProductMap from '@pages/product/product-map';
+import ProductEventMap from '@pages/product/product-event-map';
+import {EventEmitter} from 'events';
+
+export default class PackedProductsManager {
+  private eventEmitter: EventEmitter;
+
+  private entitySearchInput: EntitySearchInput;
+
+  constructor(eventEmitter: EventEmitter) {
+    this.eventEmitter = eventEmitter;
+    this.entitySearchInput = new EntitySearchInput($(ProductMap.packedProducts.searchInput), {
+      onRemovedContent: () => {
```

Comment:
if im not mistaken these can be simpliefied (brackets are not necessary when you only call one method)
```suggestion
      onRemovedContent: () => this.eventEmitter.emit(ProductEventMap.updateSubmitButtonState)
```

Reply:
I'm not very relevant on this side, so I apply directly